pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I would describe this as "you can't assign privileges you don't have"

If you attempt to create a role binding, the storage checks all of the role bindings that point to you and ensures that the permissions you currently have encompass the role the role binding your attempt to create references.

It doesn't matter if the role binding is assigning to yourself or another user, you cannot assign roles you don't already have.

Does this make sense?